wget-log

```diff
@@ -0,0 +1,5 @@
+138.68.34.161
```

Comment:
Why this file is added?

Reply:
It was automatically added whenever you run the script. I have added it to gitignore